.github/workflows/build-and-push-image-build.yml

```diff
@@ -23,26 +18,8 @@ on:
           - all
```

Comment:
Can we use a global `env` variable to store the model names to avoid having them defined in two locations?

https://stackoverflow.com/a/73274812

e.g.

```
env:
  models: ("access-om2" "access-om3")
```

and then use them in the options and later in the `generate-matrix` job?

Reply:
Unfortunately, the 'choice' type doesn't yet allow dynamic choices...only other way would be to have them input the model name or 'all' manually as a string...Not sure if thats the best idea either, but it'd be better than a static choice I suppose.

Reply:
Park it. Have made an issue to address when this is supported.

https://github.com/ACCESS-NRI/build-ci/issues/64